packages/next/lib/constants.ts

```diff
@@ -15,6 +15,9 @@ export const NEXT_PROJECT_ROOT_DIST_SERVER = join(
   'server'
 )
 
+// Regex for API routes
+export const API_ROUTE = /^\/api(?:\/.*)?$/
```

Comment:
```suggestion
export const API_ROUTE = /^\/api(?:\/|$)/
```

Nit, but I think this regex is more clear.

Reply:
This is not going to work for `/api/users`.